sdk/nodejs/iot/zMixins.ts

```diff
@@ -76,6 +76,16 @@ export class IoTHubEventSubscription extends appservice.EventSubscription<EventH
             connection: bindingConnectionKey,
         }];
 
+        pulumi.all([iotHub.fallbackRoute, iotHub.routes]).apply(([fallbackRoute, routes]) => {
+            if(fallbackRoute.enabled){
```

Comment:
Can `fallbackRoute` be `undefined` at this point?

Reply:
I'll add a check just to be sure

Reply:
fixed in 3de9fa2b988eb44505efbe34d1911977f28ef23b